sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -1179,6 +1179,19 @@ def model_bigqueryio(p, write_project='', write_dataset='', write_table=''):
       write_disposition=beam.io.BigQueryDisposition.WRITE_TRUNCATE,
       create_disposition=beam.io.BigQueryDisposition.CREATE_IF_NEEDED)
   # [END model_bigqueryio_write]
+  
+  # [START model_bigqueryio_write_dynamic_input]
+  table_names = (p | beam.Create([
+      ('error', 'my_project.dataset1.error_table_for_today'),
+      ('user_log', 'my_project.dataset1.query_table_for_today'),
+  ])
```

Comment:
it seems that you're missing an extra parenthesis here @rincon-santi . Could you add it please?

Reply:
also, if you feel up to it, the same document users table partitioning : ) - do you think you could add a snippet for that?

It would look something like this:
```
        bq_output = input
            | beam.io.WriteToBigQuery(
                (TABLE something something),
                additional_bq_parameters={'timePartitioning': {'type': 'HOUR'}})
```

That's not a must. Only if you feel up to it : )